FirebasePerformance/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * Create a random number of delay for remote config fetch during app starts.
 * Fix log spamming when Firebase Performance is disabled. (#8423, #8577)
 * Fix heap-heap-buffer overflow when decoding strings. (#8628)
+* Fix a crash related to FPRSessionDetails. (#8691)
```

Comment:
The changes above have been released. You may want to put the new changes to v8.10.0.

Reply:
Sounds good, done.

Reply:
Great, please push the commit :)

Reply:
Apologies! Now done for real!